server/main.lua

```diff
@@ -14,14 +14,15 @@ end)
 -- Deduct stored game time from player on logout
 RegisterNetEvent('qbx_vehicleshop:server:removePlayer', function(citizenid)
     if not financeTimer[citizenid] then return end
+    local finance = require 'server.finance'
 
     local playTime = financeTimer[citizenid]
-    local vehicles = FetchFinancedVehicleEntitiesByCitizenId(citizenid)
-    for _, v in pairs(vehicles) do
+    local vehicles = finance.fetchFinancedVehicleEntitiesByCitizenId(citizenid)
+    for _, v in ipairs(vehicles) do
```

Comment:
prefer a numerical for loop https://manason.github.io/effective-fivem-lua/tables/#use-numeric-for-loops-when-iterating-over-an-array